README.md

```diff
@@ -4,9 +4,9 @@
 
 ---
 
-[Key Features](#key-features) •
-[Installation](https://openvinotoolkit.github.io/training_extensions/latest/guide/get_started/installation.html) •
-[Documentation](https://openvinotoolkit.github.io/training_extensions/latest/index.html) •
+[Key Features](#key-features)
+[Installation](https://openvinotoolkit.github.io/training_extensions/1.4.1/guide/get_started/installation.html)
+[Documentation](https://openvinotoolkit.github.io/training_extensions/1.4.1/index.html)
```

Comment:
Was the "•" removed intentionally?

Reply:
oh..it's my fault. those are the separators. will revert it.